web-ui/src/streaming/introspection/IntrospectionTable.tsx

```diff
@@ -0,0 +1,128 @@
+// Browse the contents of a table or a view.
+
+import Card from '@mui/material/Card'
+import { DataGridPro, GridColumns, useGridApiRef } from '@mui/x-data-grid-pro'
+import { useQuery } from '@tanstack/react-query'
+import { useEffect, useState } from 'react'
+import { PipelineDescr, ProgramDescr, PipelineStatus, OpenAPI } from 'src/types/manager'
+import { parse } from 'csv-parse'
+import { parseProjectSchema } from 'src/types/program'
+
+export type IntrospectionTableProps = {
+  pipelineDescr: PipelineDescr | undefined
+  name: string | undefined
+}
+
+export const IntrospectionTable = ({ pipelineDescr, name }: IntrospectionTableProps) => {
```

Comment:
why "Introspection"?
Why not "browse"?

Reply:
Ok will change in next PR when I'm using leonids new APIs